CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Packaging
 
-* MapboxNavigation now requires [MapboxMaps v10.5.1](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.5.1). ([#4048](https://github.com/mapbox/mapbox-navigation-ios/pull/4048))
+* MapboxNavigation now requires [MapboxMaps v10.5._x_](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.5.1). ([#4048](https://github.com/mapbox/mapbox-navigation-ios/pull/4048))
```

Comment:
A separate v2.4.2 section should mention that the dependency resolution issue in SPM has been fixed.